trieste/acquisition/rule.py

```diff
@@ -294,3 +294,61 @@ def acquire(
         state_ = TrustRegion.State(acquisition_space, eps, y_min, is_global)
 
         return point, state_
+
+
+class BatchAcquisitionRule(AcquisitionRule[None, SearchSpace]):
+    """ Implements a acquisition rule for a batch of query points """
+
+    def __init__(self, num_query_points: int,
+                 builder: Optional[AcquisitionFunctionBuilder] = None):
+        """
+        :param num_query_points: The number of points to acquire.
+        :param builder: The acquisition function builder to use.
+            :class:`EfficientGlobalOptimization` will attempt to **maximise** the corresponding
+            acquisition function. Defaults to :class:`~trieste.acquisition.ExpectedImprovement`
+            with tag `OBJECTIVE`.
+        """
+
+        if not num_query_points > 0:
+            raise ValueError(
+                f"Number of query points must be greater than 0, got {num_query_points}"
+            )
+
+        self._num_query_points = num_query_points
+
+        if builder is None:
+            builder = ExpectedImprovement().using(OBJECTIVE)  # this is a placeholder for now
```

Comment:
We do not have a default acquisition function for batches for now. This may be resolved as soon as we have the mc-EI.

Reply:
maybe leave this without a default for this PR?

Reply:
OK - then should I replace those lines with a warning saying that for now the acq builder must be provided?

Reply:
nope, just change `Optional[AcquisitionFunctionBuilder] = None` to `AcquisitionFunctionBuilder` and remove the mention of the default from the docstring